src/lib/one-graph/cli-netlify-graph.js

```diff
@@ -545,7 +679,185 @@ const loadNetlifyGraphConfig = (siteRoot) => {
   return {}
 }
 
+const autocompleteOperationNames = async ({ netlifyGraphConfig }) => {
+  try {
+    let currentOperationsDoc = readGraphQLOperationsSourceFile(netlifyGraphConfig)
+    if (currentOperationsDoc.trim().length === 0) {
+      currentOperationsDoc = NetlifyGraph.defaultExampleOperationsDoc
+    }
+
+    const parsedDoc = parse(currentOperationsDoc)
+    const extracted = extractFunctionsFromOperationDoc(GraphQL, parsedDoc)
+
+    const { functions } = extracted
+
+    const sorted = Object.values(functions).sort((aItem, bItem) =>
+      aItem.operationName.localeCompare(bItem.operationName),
+    )
+
+    const perPage = 50
+
+    const allOperationChoices = sorted.map((operation) => ({
+      name: `${operation.operationName} (${operation.kind})`,
+      value: operation.operationName,
+    }))
+
+    const filterOperationNames = (operationChoices, input) =>
+      operationChoices.filter((operation) => operation.value.toLowerCase().match(input.toLowerCase()))
+
+    // eslint-disable-next-line n/global-require
+    const inquirerAutocompletePrompt = require('inquirer-autocomplete-prompt')
```

Comment:
are we requiring this here because it's the only place we need it in? to cut down on startup time?